analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -383,13 +384,13 @@ open class Npm(
         val nodeModulesDir = moduleDir.resolve("node_modules")
         if (!nodeModulesDir.isDirectory) return emptyList()
 
-        val searchDirs = nodeModulesDir.listFiles().filter { file ->
-            file.isDirectory && file.name.startsWith("@")
-        } + nodeModulesDir
+        val searchDirs = nodeModulesDir.listFiles(FileFilter {
```

Comment:
Consider keeping `file` instead of `it` for readability.

Reply:
Actually, for these (almost) one-liners I personally prefer to keep it shorter with `it`, esp. as directly before it now says _**File**Filter_.